addons/website/migrations/11.0.1.0/post-migration.py

```diff
@@ -79,13 +79,25 @@ def import_website_seo_redirection_data(env):
     """
     if not openupgrade.table_exists(env.cr, 'website_seo_redirection'):
         return
+    # Transfer normal redirections directly
     openupgrade.logged_query(
         env.cr, """
         INSERT INTO website_redirect
         (type, url_from, url_to, active, sequence)
         SELECT '301', wsr.origin, wsr.destination, True, 10
         FROM website_seo_redirection wsr""",
     )
+    # Find relocated pages and relocate them
+    env.cr.execute(
```

Comment:
Please, do it in only one logged query:
```python
openupgrade.logged_query(
    env.cr, """
    UPDATE website_page wp
    SET url = wsr.url_to
    FROM website_seo_redirection wsr
    WHERE wsr.relocate_controller AND wsr.url_from LIKE '/page/%'
        AND wp.url = wsr.url_from"""
)
```

Reply:
Thanks, done. I didn't know that syntax :blush: